### PR TITLE
id: 6557, comment: broken links to Welsh pages on Annuities tool

### PR DESCRIPTION
**Pull request by davidtrussler**
- three wrong links updated

### CODE REVIEW

**Comment by benbarnett** [comment]
:+1:

File: config/locales/landing_pages/annuities.cy.yml [hunk 1]
```diff
@@ -32,13 +32,13 @@ cy:
...
         - text: |
-            Blwydd-dal yw un ymysg amryw o opsiynau sydd gennych chi ar gyfer defnyddio’ch cronfa bensiwn i ddarparu incwm ymddeoliad. Gweler ein canllaw <a href="/cy/articles/rheolau-newydd-ynghylch-pensiynau-o-27-mawrth-2014-ymlaen">Opsiynau ar gyfer defnyddio’ch cronfa bensiwn</a> i gael gwybod mwy.
+            Blwydd-dal yw un ymysg amryw o opsiynau sydd gennych chi ar gyfer defnyddio’ch cronfa bensiwn i ddarparu incwm ymddeoliad. Gweler ein canllaw <a href="/cy/articles/yr-opsiynau-ar-gyfer-defnyddioch-cronfa-bensiwn">Opsiynau ar gyfer defnyddio’ch cronfa bensiwn</a> i gael gwybod mwy.
         - text: |
             Unwaith y byddwch wedi prynu blwydd-dal ni allwch newid eich meddwl – dylech gael cymorth neu gyngor cyn ymrwymo. Dysgwch ragor yn y camau nesaf ar ddiwedd yr offeryn hwn.
     section_three:
       heading: Dolenni defnyddiol
       list:
-        - url: /cy/videos/beth-yw-blwydd-dal
+        - url: /cy/videos/beth--yw-blwydd-dal
```
Comment (asmega) [inline config/locales/landing_pages/annuities.cy.yml]: hmmm. I guess it's already live

**Comment by asmega** [comment]
:+1:

File: config/locales/landing_pages/annuities.cy.yml (hunk 1 above)
Comment (davidtrussler) [inline config/locales/landing_pages/annuities.cy.yml]: Yes, that's not really very satisfactory is it - but that is the URL ...

Comment (asmega) [inline config/locales/landing_pages/annuities.cy.yml]: Yes, it's out in the wild so let it be.

**Pull request merged**